darts/models/forecasting/forecasting_model.py

```diff
@@ -412,12 +412,13 @@ def extreme_lags(
         Optional[int],
         Optional[int],
         Optional[int],
+        Optional[int],
```

Comment:
Is it really `Optional`? I have the impression that it can be defined for all the models, it will just be 0 by default (and never `None`)?